.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: bash
+
+sudo: false
+
+before_install:
+    - sudo apt-get -qq update
+    - sudo apt-get install -y realpath shellcheck
+
+script:
+    - tests/tests.sh
+    - shellcheck check-trailing-whitespaces
+    - shellcheck tests/tests.sh
```

Comment:
shouldn't we call `check-trailing-whitespaces` as well ?

Reply:
Not yet: it would report trailing whitespaces in the test fixtures files but they are there on purpose.

Reply:
you see. that's the thing. it's hard to use this until exclusions will be supported

Reply:
I know, the goal of this PR is to provide a first minimal version, not the first release.

Reply:
incremental work is nice when you could take every single incremental part and show it works
here, it's hard to say, as we don't even run the tool as it's too restrictive :(